include/access_management.js

```diff
@@ -147,7 +147,8 @@ module.exports = function(pb) {
                 'ACCESS_ADMINISTRATOR': ls.get('ACCESS_ADMINISTRATOR'),
             };
         }
-        return null;
+        // Return an empty object instead of null, so things won't break after return
+        return {};
```

Comment:
The null is here so that the code explicitly has to handle cases where it is calling the function in a deprecated manner.  Was there a specific use case where it needed the change?